industrial_ci/src/env.sh

```diff
@@ -73,11 +73,7 @@ if [ -n "$UBUNTU_OS_CODE_NAME" ]; then # for backward-compatibility
     OS_NAME=ubuntu
 fi
 
-if [ -n "$DOCKER_BASE_IMAGE" ]; then
-    # try to guess OS from default image scheme
-    OS_CODE_NAME=${OS_CODE_NAME:-${DOCKER_BASE_IMAGE##*:}} # use tag
-    OS_NAME=${OS_NAME:-${DOCKER_BASE_IMAGE%%:*}} # use repo
-else
+if [ -z "$DOCKER_BASE_IMAGE" ]; then
```

Comment:
Now with this line and the following `if` block, we're assuming that the specified Docker image contains ROS in it. Can we always assume that? E.g. what if there's anyone who wants to use certain Docker image that doesn't contain ROS?